### DISCUSSION TITLE
Is `RAILS_SERVE_STATIC_FILES` as must for static assets? Is there a plan to let Traefik to mount the extract assets dir?

### DISCUSSION THREAD

**Discussion by MikeChongCan** [basecamp/kamal#876] [Q&A] [answered]
Hello!

I think in the doc, `RAILS_SERVE_STATIC_FILES=true` is missing?

Is it a must?

Btw, do we have a plan to let Traefik to serve these assets?

**Comment by acidtib** [comment, marked as answer]
the docs dont mention `RAILS_SERVE_STATIC_FILES=true` since Kamal is not for rails only.

Traefik will be removed in Kamal version 2

**Reply by MikeChongCan** [reply]
Thanks!

So by design, `RAILS_SERVE_STATIC_FILES=true` is needed?

**Reply by MikeChongCan** [reply]
Also, for the removal of Traefik, is there any background or roadmap? Thanks!
